Monika After Story/game/script-topics.rpy

```diff
@@ -3221,6 +3224,55 @@ label monika_lovecounter_aff:
 
 default persistent._mas_last_monika_ily = None
 
+label monika_ilym_fight_start:
+    $ ilym_times_till_win = renpy.random.randint(6,10)
+    $ ilym_count = 0
+    $ ilym_quip = "I love you more, " + player + "!"
+    m 3hubfb "[ilym_quip]"
+    jump monika_ilym_fight
+
+label monika_ilym_fight:
+    $ _history_list.pop()
+    menu:
+        m "[ilym_quip]{fast}"
+        "No, I love you more!":
+            if ilym_count < ilym_times_till_win:
+                if renpy.random.randint(1,3) == 1:
+                    python:
+                        ilym_quips = [
+                            "No, I love you waaaaaaaaay more!",
+                            "No, I definitely love you more!",
+                            "Nuh-uh, I love you more!",
+                            "No no no, I love you more!"
+                        ]
+                        ilym_quip = renpy.random.choice(ilym_quips)
```

Comment:
Do a substitute here in case we add potential `[player]`s
```suggestion
                        ilym_quip = renpy.substitute(renpy.random.choice(ilym_quips))
```